.env.rc1

```diff
@@ -191,6 +214,22 @@ GCP_FORNO_SOUTHAMERICA_EAST1_FULL_NODES_DISK_SIZE=100
 GCP_FORNO_SOUTHAMERICA_EAST1_FULL_NODES_NODEKEY_DERIVATION_STRING=GCP_SOUTHAMERICA_EAST1
 # suffix of the static nodes file, e.g. `<env>.<suffix>`
 GCP_FORNO_SOUTHAMERICA_EAST1_FULL_NODES_STATIC_NODES_FILE_SUFFIX=gcp-southamerica-east1
+GCP_FORNO_SOUTHAMERICA_EAST1_FULL_NODES_RPC_API_METHODS="eth,net,rpc,web3"
+GCP_FORNO_SOUTHAMERICA_EAST1_FULL_NODES_GETH_GC_MODE="full"
+GCP_FORNO_SOUTHAMERICA_EAST1_FULL_NODES_USE_GSTORAGE_DATA=true
+
+
+# ---- Private txnodes context ----
+
+GCP_PRIVATE_TXNODES_GCP_PROJECT_NAME=celo-testnet-production
```

Comment:
Do we want to move all testnets to k8s archive nodes?

Reply:
I'd say we do not want, at least I cannot think any reason to migrate current archive nodes included with `testnet` package to `archive` nodes based on the `celo-fullnode` package.
Mainnet cluster/deployment is special as there are not validators or regular tx-nodes, so to deploy only fullnodes using `testnet` required some changes on current code and seemed more appropriate using the celo-fullnode package. But also I think both solutions seems in the end a bit artificial (also related right now we have "contexted" commands/packages and non-contexted commands/packages).